src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -241,6 +243,32 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ResolvePublishAssemblies>
 
   </Target>
+  
+ <!--
+    ============================================================
+                     RunResolvePublishAssemblies
+
+    Gets the assemblies to be copied to the publish directory
+    ============================================================
+    -->
+
+  <Target Name="ComputeAssembliesToBeFiltered"
+          Condition="'$(FilterProjFile)'!=''" >
+    <PropertyGroup>
+      <FilterProFileDir>$(MSBuildProjectExtensionsPath)\filterprofile</FilterProFileDir>
+      <FilterProjectAssetsFile>$(FilterProFileDir)\project.assets.json</FilterProjectAssetsFile>
```

Comment:
This is created in https://github.com/dotnet/sdk/pull/685/files#diff-ffd1a974928a37a20fbc471c040962acR265 as product of restore on the filterprojfile